src/modules/vtol_att_control/standard.cpp

```diff
@@ -177,28 +177,16 @@ void Standard::update_vtol_state()
 		} else if (_vtol_schedule.flight_mode == vtol_mode::TRANSITION_TO_FW) {
 			// continue the transition to fw mode while monitoring airspeed for a final switch to fw mode
 
-			const bool airspeed_triggers_transition = PX4_ISFINITE(_airspeed_validated->calibrated_airspeed_m_s)
+			const bool airspeed_triggers_transition = PX4_ISFINITE(get_filtered_airspeed())
 					&& !_params->airspeed_disabled;
 			const bool minimum_trans_time_elapsed = time_since_trans_start > getMinimumFrontTransitionTime();
 
 			bool transition_to_fw = false;
 
 			if (minimum_trans_time_elapsed) {
 				if (airspeed_triggers_transition) {
-
-					// The logic that will prevent spikes in airspeed measurements to trigger the transition
```

Comment:
The new filtered airspeed will replace this functionality for preventing spikes to trigger transition.

Reply:
This should go into the commit description (ideally title too). Same with back-transition. When I read the commit, it would be awesome to see that FT acceptence, BT acceptance and FT blending are all affected.